apiserver/bundle/bundle.go

```diff
@@ -10,20 +11,43 @@ import (
 	"github.com/juju/errors"
 	"gopkg.in/juju/charm.v6-unstable"
 
+	"github.com/juju/juju/apiserver/common"
+	"github.com/juju/juju/apiserver/facade"
 	"github.com/juju/juju/apiserver/params"
 	"github.com/juju/juju/constraints"
+	"github.com/juju/juju/state"
 	"github.com/juju/juju/storage"
 )
 
-// GetBundleChanges returns the list of changes required to deploy the given
-// bundle data. The changes are sorted by requirements, so that they can be
-// applied in order.
-func (c *Client) GetBundleChanges(args params.GetBundleChangesParams) (params.GetBundleChangesResults, error) {
```

Comment:
You can't just delete this from the client facade now. We have committed to backwards compatibility with RC1.

This means that this method needs to stay, but can forward to the new facade for implementation.